.github/ct-install.yaml

```diff
@@ -1,6 +1,6 @@
 remote: origin
 target-branch: master
-helm-extra-args: --timeout 180s
+helm-extra-args: --timeout 300s
```

Comment:
5 minutes is not an acceptable increase for a helm timeout tbh.
It would lead to timeouts on some users' systems